Changelog.md

```diff
@@ -243,4 +243,4 @@ __Bugfixes__
 
 ## OpenStudio-HPXML v0.7.0 Beta
 
-- Initial beta release
\ No newline at end of file
+- Initial beta release
```

Comment:
@shorowit Does it need to be a commit that changes results?

Reply:
I'm testing that it doesn't reflect the changes that were just made to the master branch.